libs/components/docs-tools/project.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "docs-tools",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "projectType": "library",
+  "sourceRoot": "libs/components/docs-tools/src",
+  "prefix": "sky",
+  "tags": ["component", "npm"],
+  "targets": {
+    "build": {
+      "executor": "@nx/angular:package",
+      "outputs": ["{workspaceRoot}/dist/libs/components/docs-tools"],
+      "options": {
+        "project": "libs/components/docs-tools/ng-package.json"
+      },
+      "configurations": {
+        "production": {
+          "tsConfig": "libs/components/docs-tools/tsconfig.lib.prod.json"
+        },
+        "development": {
+          "tsConfig": "libs/components/docs-tools/tsconfig.lib.json"
+        }
+      },
+      "defaultConfiguration": "production",
+      "dependsOn": [
+        "^build",
+        {
+          "projects": ["core"],
+          "target": "build"
+        }
+      ],
+      "inputs": [
+        "buildInputs",
+        "^buildInputs",
+        "{workspaceRoot}/libs/components/docs-tools/testing/src/**/*",
+        "!{workspaceRoot}/libs/components/docs-tools/testing/src/**/*.spec.ts",
+        "!{workspaceRoot}/libs/components/docs-tools/testing/src/**/fixtures/**/*"
```

Comment:
There is no `testing` subproject, right? Remove the `dependsOn` and `inputs` and just use the defaults.